bricks/template/brick.yaml

```diff
@@ -46,3 +46,13 @@ vars:
     description: Your app version
     default: "1"
     prompt: Build number?
+  json_format:
```

Comment:
I think the param should be `json_field_rename_format`. `json_format` makes me feel like JSON:API or other formats of JSON body instead.

Reply:
@luongvo Fixed in 54af0f2